colour/models/hdr_cie_lab.py

```diff
@@ -39,14 +43,60 @@
 __email__ = 'colour-science@googlegroups.com'
 __status__ = 'Production'
 
-__all__ = ['XYZ_to_hdr_CIELab', 'hdr_CIELab_to_XYZ', 'exponent_hdr_CIELab']
+__all__ = ['exponent_hdr_CIELab', 'XYZ_to_hdr_CIELab', 'hdr_CIELab_to_XYZ']
+
+
+def exponent_hdr_CIELab(Y_s, Y_abs, method='Fairchild 2011'):
+    """
+    Computes *hdr-CIELAB* colourspace *Lightness* :math:`\epsilon` exponent
+    using *Fairchild and Wyble (2010)* or *Fairchild and Chen (2011)* method.
+
+    Parameters
+    ----------
+    Y_s : numeric or array_like
+        Relative luminance :math:`Y_s` of the surround in range [0, 1].
+    Y_abs : numeric or array_like
+        Absolute luminance :math:`Y_{abs}` of the scene diffuse white in
+        :math:`cd/m^2`.
+    method : unicode, optional
+        **{'Fairchild 2011', 'Fairchild 2010'}**,
```

Comment:
To pass a method? I'm not sure to understand.

Reply:
Yeah. Alternatively, some more elaborate detection of typos (i.e,., illegal strings) further downstream might be nice.

Right now passing `Fairchild  2010` is silently going to use `Fairchild 2011`.

Reply:
Good point, I will do `if elif else raise` instead of `if else` here for now. I was indeed thinking about indeed having a more forgiving method passing mechanism, nothing implemented or tested though.

Reply:
I actually ended up checking before entering the conditional block because the `method` argument is being used in the 6 definitions.